File: clients/backend/psychopass/psychoPasses.js
class PsychoPasses {
    url;
    constructor(url = `${process.env.BACKEND_URL}/psychopass/user`) {
        this.url = url;
    }

    async get(userID) {
        try {
            const response = await fetch(`${this.url}?id=${userID}`, {
                method: "GET",
                headers: {
                    "Content-type": "application/json",
                    "User-Agent": `sibyl-discord/${process.env.npm_package_version} node.js/${process.version}`,
                    "Authorization": `Token ${process.env.BACKEND_API_KEY}`
                }
            });
            if (!response.ok) return response.status;
            return response.json();
        } catch (error) {
            console.error(error);
        }
    }
}

const psychoPasses = new PsychoPasses();
export default psychoPasses;

File: clients/backend/psycho_pass/psycho_passes.py
from os import getenv
from sys import version
from requests import get


class PsychoPasses:
    url = f"{getenv('BACKEND_URL')}/psychopass/user"

    @classmethod
    def read(cls, user_id: int) -> dict:
        response = get(
            f"{cls.url}?id={user_id}",
            headers={
                "Accept": "application/json",
                "User-Agent": f"sibyl-discord python/{version}",
                "Authorization": f"Token {getenv('BACKEND_API_KEY')}",
            },
        )

        response.raise_for_status()
        return response.json()
